src/classes/class-cachemanager.php

```diff
@@ -247,6 +254,39 @@ public function flush_fcgi_cache() {
 			return $deleted;
 		}
 
+		/**
+		 * Flush FCGI cache for posts pages and orders.
+		 */
+		public function flush_fcgi_cache_selectively_on_save( $post_id ) {
+
+			// \WooCart\Log\Socket::log( ["kind"=>"flush_fcgi_cache_selectively_on_save", "post"=>$post_id] );
+			if ( wp_is_post_revision( $post_id ) ) {
+				return;
+			}
+			$this->flush_fcgi_cache();
+
+		}
+
+		/**
+		 * Flush FCGI cache for posts pages and orders.
+		 */
+		public function flush_fcgi_cache_selectively_on_delete( $post_id ) {
+
+			$post_type = get_post_type( $post_id );
+			// \WooCart\Log\Socket::log( ["kind"=>"flush_fcgi_cache_selectively_on_delete", "post"=>$post_type] );
+			if ( 'post' === $post_type ) {
+				$this->flush_fcgi_cache();
+			}
+
+			if ( 'page' === $post_type ) {
+				$this->flush_fcgi_cache();
+			}
+
+			if ( 'product' === $post_type ) {
+				$this->flush_fcgi_cache();
+			}
```

Comment:
Maybe we can combine these three in one and adhere to the DRY principle :)

Reply:
The version needs to be bumped as well!

Reply:
I want to maintain clear coverage, for example, if I use in_array then one values covers entire block.